### ISSUE TITLE
Export declarations for Svelte and Vite not present

### ISSUE THREAD

**Issue by polvallverdu** [artiebits/svelte-seo#73]
**Describe the bug**
Svelte's Vite plugin now requires to declare the exports

**To Reproduce**

1. Install and use svelte-seo on latest SvelteKit version

**Screenshots**
```
12:20:56 PM [vite-plugin-svelte] WARNING: The following packages have a svelte field in their package.json but no exports condition for svelte.

svelte-seo@1.5.4

Please see https://github.com/sveltejs/vite-plugin-svelte/blob/main/docs/faq.md#missing-exports-condition for details.
```

**Additional context**
"@sveltejs/kit": "^2.0.6"
"@sveltejs/vite-plugin-svelte": "^3.0.1"
"vite": "^5.0.11"

I'm looking forward on opening a PR if I get some free time.

**Comment by lubiah** [comment]
@polvallverdu , try submitting a pull request if you do get some time to spare

**Comment by artiebits** [comment]
thank you for contribution, I've released svelte-seo@1.6.0
